setup.py

```diff
@@ -121,13 +121,13 @@ def read(*names, **kwargs):
         'open source', 'scan', 'license', 'package', 'dependency',
         'copyright', 'filetype', 'author', 'extract', 'licensing',
     ],
-    python_requires='>=2.7,<3',    
+    python_requires='>=2.7,<4',    
```

Comment:
We are not supporting Python 3 yet, so you need to keep this chnage in a branch of yours and I cannot merge this in develop yet